src/NServiceBus.Core.Tests/Serializers/XML/Issue_2796.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.Core.Tests.Serializers.XML
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Serializers.XML.Test;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class Issue_2796
```

Comment:
@adamralph Arguably, the system under test is the `XmlMessageSerializer`, the SerializerFactory is just a util class for testing.